tests/installation/logs_from_installation_system.pm

```diff
@@ -41,6 +41,7 @@ sub run {
             die "IPL device was not set correctly";
         }
     }
+
```

Comment:
probably unintended line break or we need to change the copyright date.

Reply:
I tested the code there in the past. I have reverted it now.

Reply:
git checkout HEAD^1 logs_from_installation_system.pm